### PR TITLE
Added refind support

### PR DESCRIPTION
**Pull request by Boria138**

### CODE REVIEW

**Comment by pastaq** [comment]
A lot of folks use refind when the manually enable dual booting so I don;t have an issue with this. This wont affect regular users in any way?

File: usr/share/device-quirks/scripts/kernel-options-manager [hunk 1]
```diff
@@ -43,12 +47,17 @@ case "$1" in
...
-             echo "Updating $entry_file"
+            echo "Updating $entry_file"
             if [ $bootloader = "systemd" ]; then
             	sed -i "/^$options/ s~\$~ $search_string~" "$entry_file"
             elif [ $bootloader = "grub" ]; then
             	sed -i "s/quiet/quiet $search_string/" "$entry_file"
-	    	    fi
+            elif [ $bootloader = "refind" ]; then
+                if grep -qF "rw" "$entry_file"
+                then sed -i "s/rw/rw $search_string/" "$entry_file"
```
Comment (pastaq) [inline usr/share/device-quirks/scripts/kernel-options-manager:57]: Can you ensure consistent styling here?

File: usr/share/device-quirks/scripts/kernel-options-manager [hunk 2]
```diff
@@ -68,7 +77,10 @@ case "$1" in
     --list)
         # Print the current kernel parameters
         echo "Current kernel parameters:"
-        grep "^$options" "$entry_file" | sed 's/^"'$options'" //' | sed 's/\s*$//' | tr ' ' '\n'
+        if [ "$bootloader" = "refind" ]
+        then grep "^\"$options\"" "$entry_file" | sed 's/^\"'"$options"'" //' | sed 's/\s*$//' | tr ' ' '\n'
```
Comment (pastaq) [inline usr/share/device-quirks/scripts/kernel-options-manager:81]: Also here.

**Review by pastaq — CHANGES_REQUESTED** [review]
Minor readability/style fixes.

**Comment by alkazar** [comment]
Not sure I understand the purpose of this. Maybe I don't understand refind. I thought it just basically chainloads. Why does it need to have its own kernel parameters?

**Comment by pastaq** [comment]
LGTM, and if it has no negative effect on others then what's the problem?

from the arch wiki:
```
After installing rEFInd's files to the ESP, verify that rEFInd has created refind_linux.conf containing [kernel parameters](https://wiki.archlinux.org/title/Kernel_parameters) in the same directory as your kernel. This configuration file will not be created if you used the --usedefault option, run mkrlconf as root to create it.
Warning: When refind-install is run in chroot (e.g. in live system when installing Arch Linux) /boot/refind_linux.conf is populated with kernel options from the live system not the one on which it is installed. Edit /boot/refind_linux.conf and make sure the [kernel parameters](https://wiki.archlinux.org/title/Kernel_parameters) in it are correct for your system, otherwise you could get a kernel panic on your next boot. See [#refind_linux.conf](https://wiki.archlinux.org/title/REFInd#refind_linux.conf) for an example file.
```

**Review by pastaq — APPROVED** [review]

**Comment by alkazar** [comment]
There is an upcoming refactor that will also cover this case.

**Pull request closed without merging**